.github/workflows/bench.yml

```diff
@@ -80,19 +80,26 @@ jobs:
 
       - name: Profile client/server transfer
         run: |
-          { mkdir server; \
-            cd server; \
-            taskset -c 0 nice -n -20 \
-            cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
-              --bin neqo-server -- --db ../test-fixture/db "$HOST:4433" || true; } &
-          mkdir client; \
-            cd client; \
-            time taskset -c 1 nice -n -20 \
-            cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
-              --bin neqo-client -- --output-dir . "https://$HOST:4433/$SIZE"
-          killall -INT neqo-server
-          cd ${{ github.workspace }}
-          [ "$(wc -c < client/"$SIZE")" -eq "$SIZE" ] || exit 1
+          for CC in newreno cubic; do
+            { mkdir -p server; \
+              cd server; \
+              taskset -c 0 nice -n -20 \
+              cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
+                --bin neqo-server -- --db ../test-fixture/db --cc "$CC" "$HOST:4433" || true; } &
+            mkdir -p client; \
+              cd client; \
+              time taskset -c 1 nice -n -20 \
+              cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
+                --bin neqo-client -- --output-dir . --cc "$CC" "https://$HOST:4433/$SIZE"
+            killall -INT neqo-server
+            cd ${{ github.workspace }}
+            [ "$(wc -c < client/"$SIZE")" -eq "$SIZE" ] || exit 1
+            while [ ! -f server/flamegraph.svg ]; do sleep 1; done
+            for PEER in client server; do
+              mv "$PEER/flamegraph.svg" "$PEER-$CC.svg"
+              mv "$PEER/perf.data" "$PEER-$CC.perf"
```

Comment:
You might also use `-o` with perf to set where this file goes.  Though that means moving that option out of `$PERF_CMD`